app/models/concerns/act_as_taggable_on.rb

```diff
@@ -1,25 +1,21 @@
 module ActAsTaggableOn
   def acts_as_taggable_on
     class_eval do
+      require "hair_trigger"
+
       def self.tagging_relation_name
-        "#{name.underscore}_tags".to_sym
+        "taggings".to_sym
```

Comment:
Code in topological_inventory-api references the `tagging_relation_name` so this is for compatibility until that is changed, then we can just use a normal `has_many :taggings` down here https://github.com/ManageIQ/topological_inventory-core/pull/103/files#diff-2441abf8e311a53a22e85595c554e4a2R10